examples/example_wip_jung_hackz.py

```diff
@@ -0,0 +1,194 @@
+"""Example for individual address write."""
+import sys
+sys.path.append('./')
+
+import logging
+import asyncio
+
+from xknx import XKNX
+from xknx.prog import ProgDevice
+from xknx.prog.management import NM_EXISTS, NM_OK, NM_TIME_OUT, NetworkManagement
+from xknx.telegram.address import IndividualAddress, GroupAddress
+from xknx.telegram import Telegram
+
+
+NM_OK = 0
+NM_EXISTS = 1
+NM_TIME_OUT = 2
+
+class UltraHacker:
+    """Class for network management functionality."""
+
+    def __init__(self, xknx: XKNX):
+        """Construct NM instance."""
+        self.xknx = xknx
+        xknx.telegram_queue.register_telegram_received_cb(self.telegram_received_cb)
+        # map for registered devices
+        self.reg_dev: dict[IndividualAddress, Device] = {}
```

Comment:
Device doesnt exist here.

I guess you mean:

```suggestion
        self.reg_dev: dict[IndividualAddress, ProgDevice] = {}
```